.github/workflows/tsc.yml

```
@@ -1,13 +1,15 @@
-    branches: [ main ]
+    branches: [main]
 jobs:
   TypeScriptCompiler:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Install modules
-      run: yarn
-    - name: Run TypeScriptCompiler
-      run: yarn check-types
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)